src/firefox/index.js

```diff
@@ -193,6 +193,7 @@ export type ConfigureProfileOptions = {
 
 export type ConfigureProfileFn = (
   profile: FirefoxProfile,
+  customs: {},
```

Comment:
So this is the last step in `configureProfile`, which I haven't properly tested yet.
```
 Object.keys(customPrefs).forEach((custom) => {
  profile.setPreference(custom, customPrefs[custom]);
});
```
Is using a defined flow type and yargs parsing of values suffice for control over what properties user assigns? At the moment command happily takes any property name (and number of them)